docs/Blog-Post-Submission-Guide.md

```diff
@@ -0,0 +1,86 @@
+# Blog Post Contributor Guide
+
+This section describes the guidelines for contributing new blog posts to the
+Kata Containers website.
+
+## Share your stories on the StarlingX Website
+
+Are you experimenting with Kata Containers or have it deployed in production and
+would like to share your story as a case study? Do you have a use case that
+Kata Containers can make more secure, but the world doesn't know it yet? Do you
+have features in the runtime that you like and would like to highlight? Do you
+have a Kata Containers demo that you would like to draw attention to?
+
+Share your Kata Containers story on the [Kata Containers blog](https://www.katacontainers.io/blog/)!
+You are only a few steps away...
+
+### Kata Containers website source
+
+Like the rest of the Kata Containers artifacts, the project’s website code and
+content are stored in a [GitHub repository](https://github.com/kata-containers/www.katacontainers.io).
+
+The blog posts are written using markdown language that is mainly plain text
+with a few easy formatting conventions to create lists, add images or code blocks,
+or format the text.
+
+You can find many [cheat sheets](https://www.markdownguide.org/cheat-sheet/)
+floating on the web to get in terms of the basic syntax. You can also check the
+[source files of the already existing blog posts](https://github.com/kata-containers/www.katacontainers.io/tree/main/src/pages/blog),
+where you will find examples of all the basic items that you will need for your
+new entry.
+
+### Create a new blog post
+
+When you create a new blog post, you need to create a new file in the
+[’src/pages/blog/' folder](https://github.com/kata-containers/www.katacontainers.io/tree/main/src/pages/blog)
+with a '.md' extension.
+
+The markdown file has a few formatting conventions in its header to capture the
+title, author, publishing date and category of your blog post.
+
+The header looks like the following:
+
+::
+
+  ---
+  templateKey: blog-post
+  title: The Title of Your Amazing Blog Post
+  author: Your Name
+  date: 2021-01-28T16:23:52.741Z
+  category:
+    - value: category-6-wjkXzEM2
+      label: Features & Updates
+  ---
```

Comment:
Should this be in triple backticks so it shows up as code properly?

Reply:
Fixed now.